### PR TITLE
Add mailmap entry

### PR DESCRIPTION
**Pull request by maxnoe**
### Description

This pull request is to address my name and related email change

### Checklist for package maintainer(s)

This checklist is meant to remind the package maintainer(s) who will review this pull request of some common things to look for. This list is not exhaustive.

- [ ] Do the proposed changes actually accomplish desired goals?
- [ ] Do the proposed changes follow the [Astropy coding guidelines](https://docs.astropy.org/en/latest/development/codeguide.html)?
- [ ] Are tests added/updated as required? If so, do they follow the [Astropy testing guidelines](https://docs.astropy.org/en/latest/development/testguide.html)?
- [ ] Are docs added/updated as required? If so, do they follow the [Astropy documentation guidelines](https://docs.astropy.org/en/latest/development/docguide.html#astropy-documentation-rules-and-guidelines)?
- [ ] Is rebase and/or squash necessary? If so, please provide the author with appropriate instructions. Also see ["When to rebase and squash commits"](https://docs.astropy.org/en/latest/development/when_to_rebase.html).
- [ ] Did the CI pass? If no, are the failures related? If you need to run daily and weekly cron jobs as part of the PR, please apply the `Extra CI` label. Codestyle issues can be fixed by the [bot](https://docs.astropy.org/en/latest/development/workflow/development_workflow.html#pre-commit).
- [ ] Is a change log needed? If yes, did the change log check pass? If no, add the `no-changelog-entry-needed` label. If this is a manual backport, use the `skip-changelog-checks` label unless special changelog handling is necessary.
- [ ] Is this a big PR that makes a "What's new?" entry worthwhile and if so, is (1) a "what's new" entry included in this PR and (2) the "whatsnew-needed" label applied?
- [ ] Is a milestone set? Milestone must be set but `astropy-bot` check might be missing; do not let the green checkmark fool you.
- [ ] At the time of adding the milestone, if the milestone set requires a backport to release branch(es), apply the appropriate `backport-X.Y.x` label(s) *before* merge.

### CODE REVIEW

**Comment by pllim** [comment]
Now sure how to milestone this... I'll let @astrofrog deal with it as a release manager. Thanks!

**Comment by maxnoe** [comment]
Would be nice to have it in the all future release, so I guess the milestone should be LTS to get it backported onto all active branches?

**Comment by maxnoe** [comment]
@astrofrog you added a thumbs up to my comment but milestoned to 5.2, shouldn't it be 5.0?

**Comment by astrofrog** [comment]
I couldn't find the right labels on my phone for backporting to LTS, will try on computer later unless someone beats me to it

**Comment by pllim** [comment]
I updated milestone and added label. Will leave it to Tom R to approve/merge. Thanks!

**Comment by maxnoe** [comment]
Too bad this didn't make it into 5.2....

 @astrofrog merge?

**Review by nstarman — APPROVED** [review]

**Comment by pllim** [comment]
I really don't know what is the mailmap policy. Usually it is automatically generated during release, so while I don't have objection, I would like @astropy/astropy-project-release-team to approve this. Thanks for your patience!

File: .mailmap
Comment (bsipocz) [inline .mailmap:204]:
```suggestion
Maximilian Linhoff           <maximilian.linhoff@tu-dortmund.de>  <maximilian.noethe@tu-dortmund.de>
```

**Review by bsipocz — APPROVED** [review]

**Comment by bsipocz** [comment]
> I really don't know what is the mailmap policy

imo, name fixes coming directly from the affected contributors should always be honoured without questions and go in asap.

**Comment by bsipocz** [comment]
@maxnoe - you may want to open follow-up PRs that fixes your name in the website repo, etc. While some of those lists are auto generated during release, I don't see any reasons why it should not be fixed manually for the interim.

**Pull request merged**
